sdk/nodejs/tests/runtime/langhost/cases/009.invoke/index.js

```diff
@@ -6,12 +6,22 @@ let pulumi = require("../../../../../");
 let args = {
     a: "hello",
     b: true,
-    c: [ 0.99, 42, { z: "x" } ],
+    c: [0.99, 42, { z: "x" }],
     id: "some-id",
     urn: "some-urn",
 };
-let result = pulumi.runtime.invoke("invoke:index:echo", args);
-result.then((v) => {
+
+let result1 = pulumi.runtime.invoke("invoke:index:echo", args);
+for (const key in args) {
+    assert.deepEqual(result1[key], args[key]);
+}
```

Comment:
demonstrates that the value comes back synchronously with properties directly on the object.